task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');
+
+    useEffect(() => {
+      fetch('https://new.world.com/fleet/121')
+        .then(response => response.json())
+        .then(json => {
+          setFeul(json);
+          console.log(json);
+        })
+    });
+
+    useEffect(() => {
+      if (!fuel) setAlertTxt('Processing...'); else
+      if (fuel.litres > 0) setAlertTxt('Need to buy more fuel'); else

```

Comment:
```suggestion
      if (fuel.litres > 10000) setAlertTxt('Need to buy more fuel'); else
```
It should be 10000, not 0. Even better, don't use magic numbers, extract this value to a constant and reference it here.